src/contexts/APIContext/types.ts

```diff
@@ -18,7 +19,8 @@ export interface ContextValues {
   getAccountId?: (publicKey: string) => Promise<false | IGetAccountIdResult>;
   getBalance?: (account: string) => Promise<false | IGetBalanceResult>;
   getMyTxs?: (account: string) => Promise<false | IGetBlockchainTransactionResult>;
-  getBlocks?: (firstIndex: number, lastIndex: number) => Promise<false | IGetBlocksResult>;
+  getBlocks?: (firstIndex: number, lastIndex: number, includeTransactions: boolean) => Promise<false | IGetBlocksResult>;
+  getBlock?: (height: number, includeTransactions: boolean) => Promise<false | IBlock>;
```

Comment:
it may need to be extended if the api always responds with the IBaseAPIResult properties.

this wasn't caught when deving because the API call is still returning an `any` which means that `getBlock` is returning an promise<any>

i'll attempt to address this as best i can with the api error response handling changes, since it's all related and implement a type that api returns so that we have to update that type for each api result added.

Reply:
How are you tracking that action item to ensure it doesn't get dropped?

Reply:
see issue #114